health/health.go

```diff
@@ -37,8 +37,13 @@ type Client struct {
 
 // NewClient creates a new instance of Client with a given app url
 func NewClient(name, url string) *Client {
```

Comment:
I wonder if it would be better to remove the original constructors to force the consideration of clienter reuse, at the expense of the convenience? 🤔

Reply:
yes, that would be the way to make it clear that we want to share clienters among different clients.